### PR TITLE
Cleaner version of #320

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: build-farm/jdk10u/Linux/x64/hotspot/setup.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+################################################################################
+
+################################################################################
+#
+# Setup for JDK 10 builds on AdoptOpenJDK build farm nodes
+#
+################################################################################
+
+[ -r /opt/rh/devtoolset-2/root/usr/bin/gcc ] && export CC=/opt/rh/devtoolset-2/root/usr/bin/gcc
+[ -r /opt/rh/devtoolset-2/root/usr/bin/g++ ] && export CXX=/opt/rh/devtoolset-2/root/usr/bin/g++
```
Comment (gdams) [inline build-farm/jdk10u/Linux/x64/hotspot/setup.sh]: do we need this in separate files? this is a universal setup command for all linux x64 builds. It's also the same as `build-farm/jdk10u/Linux/x64/openj9/setup.sh`.

File: build-farm/jdk10u/Linux/x64/openj9/setup.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# limitations under the License.
+################################################################################
+
+################################################################################
+#
+# Setup for JDK 10 builds on AdoptOpenJDK build farm nodes
+#
+################################################################################
+
+export PATH=/opt/rh/devtoolset-2/root/usr/bin:$PATH
```
Comment (gdams) [inline build-farm/jdk10u/Linux/x64/openj9/setup.sh]: essentially a duplicate of `build-farm/jdk10u/Linux/x64/hotspot/setup.sh`

File: build-farm/jdk8u/Linux/x64/openj9/setup.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# limitations under the License.
+################################################################################
+
+################################################################################
+#
+# Setup for JDK 8 builds on AdoptOpenJDK build farm nodes
+#
+################################################################################
+
+export PATH=/opt/rh/devtoolset-2/root/usr/bin:$PATH
```
Comment (gdams) [inline build-farm/jdk8u/Linux/x64/openj9/setup.sh]: again a duplicate of `build-farm/jdk10u/Linux/x64/hotspot/setup.sh`

File: build-farm/jdk10u/Linux/x64/hotspot/setup.sh (hunk 1 above)
Comment (johnoliver) [reply build-farm/jdk10u/Linux/x64/hotspot/setup.sh]: Yeah I was not a fan of the duplication. My issue was I did not feel it was clear what configuration would be built with when it was unified. I have reverted it to a single file which can be seen at: https://github.com/johnoliver/openjdk-build/blob/overhaul-build-scripts-2/build-farm/set-platform-specific-configurations.sh will get @karianna to merge that version into this PR

Comment (johnoliver) [reply build-farm/jdk10u/Linux/x64/hotspot/setup.sh]: Have now fixed this to remove the duplication so there is a file per OS, it looks fairly reasonable

**Comment by karianna** [comment]
Closing this as @johnoliver will open a fresh one with squashed commits.

**Pull request closed without merging**